src/DTS/Repository/DocDataRepository.php

```diff
@@ -22,6 +23,10 @@ public function save(DocData $docData) {
     $this->persistence->persist($docData->toArray());
   }
 
+  public function getByDocDataId($dataId) {
```

Comment:
should use a return type here

Reply:
Normally I would, the only reason I didn't is none of the other methods in the class do and I was looking to be consistent